meta/meta_test.go

```diff
@@ -327,36 +339,42 @@ func (s *testSuite) TestDDL(c *C) {
 	err = t.UpdateDDLJob(0, job, true)
 	c.Assert(err, IsNil)
 
-	err = t.UpdateDDLReorgStartHandle(job, 1)
+	// There are 3 meta key relate to index reorganization:
+	// start_handle, end_handle and physical_table_id.
+	// Only start_handle is initialized.
+	err = t.UpdateDDLReorgStartHandle(job, kv.IntHandle(1))
```

Comment:
Why not use `newHandle(1)`?

Reply:
This is related to the test below. Old-version TiDB does not set `physical_table_id`, and it always returns `maxInt64` as `end_handle`, so the type of these handles should be `Int`.